Dockerfile

```diff
@@ -13,9 +13,9 @@
 # limitations under the License.
 
 FROM golang:1.13-alpine as builder
-ADD . /usr/src/network-resources-injector
-RUN apk add --update --virtual build-dependencies build-base linux-headers && \
-    cd /usr/src/network-resources-injector && \
+COPY . /usr/src/network-resources-injector
```

Comment:
@martinkennelly I have seen people using ADD and COPY to achieve the same copy task., but don't know which one is better over the other in a certain situation.  which tool are you using to check the dockerfile semantics?

Reply:
I used hadolint. It will also inform you why something is incorrect. You can use snyk also.
https://github.com/hadolint/hadolint